README.md

```diff
@@ -29,56 +29,25 @@ Description of the Erica connector:
 More information can be found here (German):  
 [http://hc-ddr.hucki.net/wiki/doku.php/z9001/erweiterungen/s3004](http://hc-ddr.hucki.net/wiki/doku.php/z9001/erweiterungen/s3004)
 
-A schematic of our Arduino based interface can be found [on EasyEDA](https://easyeda.com/sirexeclp/erikaarduinointerface).
-![Schematic](docs/Schematic_ErikaArduinoInterface.png)
+A schematic of our Raspberry Pi based interface can be found [on EasyEDA](https://easyeda.com/sirexeclp/erikaraspberrypiinterface).
+![Schematic](docs/Schematic_ErikaRaspberryPiInterface.png)
```

Comment:
If I open the link, the picture is already a newer one, containing the full port mapping, not just the RX-to-TX mapping! 

Please update the referrenced image with the most recent https://image.easyeda.com/histories/b68a3ef1c3094ff1b3f330826ef3d6e9.png

Or maybe, this view is not showing me the whole truth?
https://github.com/Chaostreff-Potsdam/erika3004/pull/59/files?short_path=04c6e90#diff-04c6e90faac2675aa89e2176d2eec7d8

Reply:
@ArchibaldBienetre So do you still see an issue here? I am a bit confused because of your writing.

Reply:
Yes, the image is still old.

@sirexeclp may have wanted to update the image as a separate step - but let's do it here!
https://github.com/Chaostreff-Potsdam/erika3004/issues/44

Reply:
It was still not clear to me which image you meant. After taking a closer look, you probably mean that @sirexeclp updated the link into our internal docs to `docs/Schematic_ErikaRaspberryPiInterface.png`, but did not provide an updated image there in order to replace `Schematic_ErikaArduinoInterface.png`?

Because the links to EasyEDA are fine, no?

Reply:
Yes, exactly!